js/src/logger.ts

```diff
@@ -1764,11 +1762,11 @@ class ObjectFetcher<RecordType> {
   }
 }
 
-export interface EvalCase<Input, Expected> {
+export type BaseMetadata = object;
```

Comment:
out of curiosity, why `object` instead of `Record<string, unknown>`? The latter seems to compile for me.

Reply:
so that the right half (i.e. `unknown`) can be typed differently for each key

Reply:
I think `Record<string, unknown>` allows this: https://www.typescriptlang.org/play?ssl=5&ssc=2&pln=1&pc=1#code/MYewdgzgLgBAZiEAuGAlApqATgEwDzRYCWYA5gDQwCuYA1mCAO5gB8MAvDAN4BQM-MAIYoAnCPJ8BAIxQAiQVOCyJAmMBRQsVdBIC+QA

Reply:
Ok will update!